sqlmesh/dbt/loader.py

```diff
@@ -124,59 +126,65 @@ def _load_audits(
     ) -> UniqueKeyDict[str, Audit]:
         audits: UniqueKeyDict = UniqueKeyDict("audits")
 
-        project = self._load_project()
-        context = project.context
+        for project in self._load_projects():
+            context = project.context
 
-        logger.debug("Converting audits to sqlmesh")
-        for package in project.packages.values():
-            context.set_and_render_variables(package.variables, package.name)
-            for test in package.tests.values():
-                logger.debug("Converting '%s' to sqlmesh format", test.name)
-                audits[test.name] = test.to_sqlmesh(context)
+            logger.debug("Converting audits to sqlmesh")
+            for package in project.packages.values():
+                context.set_and_render_variables(package.variables, package.name)
+                for test in package.tests.values():
+                    logger.debug("Converting '%s' to sqlmesh format", test.name)
+                    audits[test.name] = test.to_sqlmesh(context)
 
         return audits
 
-    def _load_project(self) -> Project:
-        if self._project:
-            return self._project
-
-        target_name = self._context.gateway or self._context.config.default_gateway
+    def _load_projects(self) -> t.List[Project]:
+        if not self._projects:
```

Comment:
I know it has been there before but I feel like we should just use `@cached_property` and get rid of an extra level nestedness